### PR TITLE
feat(Checkbox): enhance appearance and a11y

### PR DESCRIPTION
**Pull request by MkMan**
2 main changes:
1. improve the appearance of the checkbox
2. improve accessibility by adding focus style

The changes have been checked with Liz from the UX team

checkboxes

### CODE REVIEW

File: src/components/Checkbox/Checkbox.module.css [hunk 1]
```diff
@@ -2,35 +2,31 @@
...
+  width: 15px;
+  height: 15px;
+  flex: 0 0 15px;
+  align-items: center;
+  justify-content: center;
+  border-width: 1px;
+  border-style: solid;
+  margin: 0;
   appearance: none;
   background-clip: content-box;
```
Comment (cyrusxzw) [inline src/components/Checkbox/Checkbox.module.css:20]: I guess we can remove `background-clip: content-box;`. Cause checkbox doesn't have background-image and background-color is transparent. `background-clip` will not have a visual effect.

Comment (MkMan) [reply src/components/Checkbox/Checkbox.module.css:20]: Great pick up, thank you! Let me give it a try 👍

Comment (MkMan) [reply src/components/Checkbox/Checkbox.module.css]: Done 👍

**Review by cyrusxzw — APPROVED** [review]

File: src/components/Checkbox/Checkbox.module.css [hunk 2]
```diff
@@ -46,20 +41,37 @@
...
-      top: -6px;
-    }
-    @media (--viewport-md-only) {
-      top: -7px;
+    background-color: currentcolor;
+    transform: scale(1);
+
+    @media (forced-colors: active) {
+      /* Windows High contrast mode https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#system_colors */
+      background-color: CanvasText;
```
Comment (kevin-ho87) [inline src/components/Checkbox/Checkbox.module.css:61]: never seen this before 👀

**Review by kevin-ho87 — APPROVED** [review]

File: src/components/Checkbox/Checkbox.module.css (hunk 2 above)
Comment (MkMan) [reply src/components/Checkbox/Checkbox.module.css:61]: Yeah same, I found out about it from this [page](https://moderncss.dev/pure-css-custom-checkbox-style/#step-3-styling-checked-vs-unchecked-state). Then I did some digging to understand what `CanvasText` is and found the link I put in the comment.

**Pull request merged**